### DISCUSSION TITLE
How to implement arara

### DISCUSSION THREAD

**Discussion by onekk** [James-Yu/LaTeX-Workshop#3119] [Q&A]
### Disable all the other extensions except for LaTeX Workshop, and check that you still see this issue. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes

## Describe the bug [Required]

How to implement arara

it will suffice to launch arara -v filename_with_extension

I've tryed to put this lines in the settings.json file

```

{
            "name": "arara",
            "tools": [
                "arara"
            ],
            "args": [
                "-v",
                "%DOCFILE%"
              ]
        },
```
At first place

but i obtain this error

Skipping undefined tool "arara" in recipe "arara."

Admit it somewhat obscure, I don't find a way to do a better tracing.

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.
